opl-secret-ballot/Makefile

```diff
@@ -0,0 +1,39 @@
+ROOT_DIR := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+
+DOCKER=docker run --rm -it --hostname=opl -u `id -u`:`id -g` -e HISTFILESIZE=0 -e HOME=/src --network=host -w /src -v $(ROOT_DIR):/src node:lts
```

Comment:
I think just running this directly makes more sense than forcing ourselves into a Dockerfile.

cc @matevz

Reply:
I might add that we should have building and testing consistent across all examples so we can put this into CI and have reproducibility.

I agree with @aefhm and I'm leaning more towards Docker-agnostic Makefiles. I would add a switch (via env variable?) somewhere in common.mk, which runs the Docker beforehand and then the normal Makefile rule inside it.

Reply:
https://github.com/oasisprotocol/playground/issues/16